website/docs/connect-data/concepts/Datasource-Environments.md

```diff
@@ -31,12 +31,14 @@ This allows you to create queries and switch between environments, with the quer
 
 ## Environments on Appsmith
 
-Appsmith provides two environments:
+Appsmith provides two default environments: production and staging. However, with the enterprise plan, you have can create custom environments according to your requirements.
```

Comment:
Please check upon the casing for production, staging and custom. It looks odd in the docs, as here they are mentioned as normal text and then Bold and highlighted just below.